src/services/rating_service.py

```diff
@@ -184,8 +184,8 @@ def create_rating_objects(model_response: str, vacancy_id: UUID, applicant_id: U
     list_of_rating_responses = extract_ratings_from_response(model_response)
     ratings = []
 
-    with VacancyMapper() as vacancy_mapper:
-        vacancy = vacancy_mapper.get_by_id(vacancy_id)
+    vacancy_mapper = VacancyMapper()
+    vacancy = vacancy_mapper.get_by_id(vacancy_id)
```

Comment:
Why did you decide against using pythons context manager? I´m just curious 😄

Reply:
The problem was, that I haven't implemented the __enter__ and __exit__ methods inside the abstract mongodb mapper class. They seem to be mandatory for the context manager . I updated the abstract class so we can use the same approach for both mappers